### PR TITLE
Generate builtin json getter with python script

### PR DESCRIPTION
**Pull request by take-cheeze**

### CODE REVIEW

File: configs/CMakeLists.txt [hunk 1]
```diff
@@ -1,16 +1,17 @@
 get_filename_component(CHAINER_COMPILER_ROOT_DIR ${CMAKE_CURRENT_SOURCE_DIR} PATH)
 
 set(jsons
-  chxvm.json
-  chxvm_test.json
-  dldt.json
-  ngraph.json
-  nvrtc.json
-  tvm.json
+  chxvm
```
Comment (shinh) [inline configs/CMakeLists.txt]: I think I may put a new target before this element, so.. (please check the next comment)

File: configs/gen_load_json_from_name.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+import sys
+
+
+def main(argv):
+    print('#include <configs/json_repository.h>')
+    print('#include <common/log.h>')
+    print('namespace chainer_compiler {')
+    print('namespace builtin_configs {')
+    for j in argv[1:]:
```
Comment (shinh) [inline configs/gen_load_json_from_name.py]: Could you move 'chxvm' to the front?

```
>>> i = argv.index('chxvm')
>>> [argv[i]] + argv[:i] + argv[i+1:]
```

File: configs/CMakeLists.txt (hunk 1 above)
Comment (take-cheeze) [reply configs/CMakeLists.txt]: Good point. I'm feared about it too.

File: configs/gen_load_json_from_name.py [hunk 3]
```diff
@@ -4,6 +4,9 @@
 
 
 def main(argv):
+    argv.remove('chxvm')
```
Comment (shinh) [inline configs/gen_load_json_from_name.py]: Ah, this is easier than what I suggested :)

**Pull request merged**